### PR TITLE
GEOPY-757: Add Ground TEM large-loop to geoapps

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-757 - Add Ground TEM (large-loop) option with runtest**

### CODE REVIEW

File: geoapps/inversion/components/factories/receiver_factory.py [hunk 1]
```diff
@@ -57,7 +57,13 @@ def concrete_object(self):
         elif "tdem" in self.factory_type:
             from SimPEG.electromagnetics.time_domain import receivers
 
-            return receivers.PointMagneticFluxTimeDerivative
+            if self.params.data_units == "dB/dt (T/s)":
+                return receivers.PointMagneticFluxTimeDerivative
+            elif self.params.data_units == "B (T)":
+                return receivers.PointMagneticFluxDensity
```
Comment (benk-mira) [inline geoapps/inversion/components/factories/receiver_factory.py:63]: What about H:
image

**Review by benk-mira — COMMENTED** [review]
I think you might need to handle an extra case in the receiver factory, have a look at the comment there..

File: geoapps/inversion/components/factories/receiver_factory.py (hunk 1 above)
Comment (domfournier) [reply geoapps/inversion/components/factories/receiver_factory.py:63]: The next else returns a PointMagneticField

Comment (domfournier) [reply geoapps/inversion/components/factories/receiver_factory.py:63]: The next else returns a PointMagneticField receiver...

**Review by benk-mira — APPROVED** [review]

**Pull request merged**